### ISSUE TITLE
pytoml strips incorrectly strips leading newlines from single-line strings

### ISSUE THREAD

**Issue by mosteli** [avakar/pytoml#58]
See below:
```
>>> toml_content
u'["a.com"]\nusername = "\\n0"\npassword = "0"\n'
>>> print(toml_content)
["a.com"]
username = "\n0"
password = "0"

>>> pytoml.loads(toml_content)
{u'a.com': {u'username': u'0', u'password': u'0'}}
```
The leading `\n` in `username` is stripped in the `pytoml.loads` function.

**Comment by avakar** [comment]
Thank you for the issue, it's fixed and uploaded to PyPi as version 0.1.21

**Comment by mosteli** [comment]
Thanks!
